src/arrayMethodJoin.js

```diff
@@ -4,8 +4,27 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    if (this[0] === null) {
+      this[0] = '';
+    }
+
+    let result = '' + this[0];
+    const workSepar = '' + separator;
```

Comment:
we do not like separs, please fix comments

Reply:
done